src/ucp/api/ucp.h

```diff
@@ -863,10 +863,10 @@ ucs_status_t ucp_rmem_ptr(ucp_ep_h ep, void *remote_addr, ucp_rkey_h rkey,
  *                          that the call-back is only invoked in a case when
  *                          the operation cannot be completed in place.
  *
- * @return UCS_OK           - The send operation was completed completed
- *                          immediately.
+ * @return UCS_OK           - The send operation was completed immediately.
  * @return UCS_PTR_IS_ERR(_ptr) - The send operation failed.
- * @return otherwise        - Operation was scheduled for send. The request handle
+ * @return otherwise        - Operation was scheduled for send or was already
+ *                          completed (after scheduling). The request handle
```

Comment:
If it was completed in place we return UCS_OK. I think it is ok just to state that the operation was "scheduled". Obviously, it can be potentially complemented any moment after scheduling. 

@alinask - Can you elaborate what do you want to clarify here ? Thanks